frontend/src/pages/start/main/Main.module.scss

```diff
@@ -1,102 +1,221 @@
-@import 'styles/colors';
-@import 'styles/spacings';
-@import 'styles/typography';
-@import 'styles/animations';
-@import 'styles/rules';
-
-.emulatorDialog {
-  pre {
-    margin-top: $spacing-base;
-  }
-}
-
-.loaderWrapper {
-  height: 100vh;
-}
+@import "styles/colors";
+@import "styles/spacings";
+@import "styles/typography";
+@import "styles/animations";
+@import "styles/rules";
 
 .container {
-  background: $dark-blue;
-  padding: $spacing-base;
-  color: $black;
   display: flex;
-  height: 100vh;
-  flex-direction: column;
-
-  justify-content: center;
-  align-items: center;
-
-  img {
-    width: 108px;
-    margin-bottom: 50px;
-  }
-
-  .header {
-    text-align: center;
-    margin-bottom: $spacing-l * 3;
-
-    h1 {
-      color: $color-white;
-      opacity: 0.74;
-      font-family: $font-family-days-one;
-      font-size: 35px;
-      font-weight: normal;
-      letter-spacing: 1.4px;
-      margin-bottom: $spacing-s;
+  flex-direction: row;
+  color: $light-grey;
+
+  aside {
+    width: 512px;
+    height: 100vh;
+    background-color: $background-highlight-2;
+
+    display: flex;
+    flex-direction: column;
+
+    .sideBarHeader {
+      box-sizing: border-box;
+      height: 191px;
+      width: 100%;
+      padding: 38px 53px;
+
+      display: flex;
+      flex-direction: row;
+      justify-content: flex-start;
+      align-items: flex-end;
+
+      span {
```

Comment:
Do you think it's better that we always use explicit class names instead of referencing html elements? I'm thinking that it could be more type-safe. Because now we could change `span` element to `div` in React, but forgot to also change this scss selector to `div { ... }`.

While if we are using class names there is no need to change the scss code when dom content changes.

Idk, lemme know your thoughts.

Reply:
Idk usually if you change span to div, you are doing changes that require changing the style anyway. It is still probably better to use class names.